_posts/francois_gerthoffert/2017-08-21-Infrastructure-procurement-in-large-scale-genomic-projects.md

```diff
@@ -0,0 +1,163 @@
+
+# Defining a purchase strategy
+
+The Cancer Genome Collaboratory is a research project involving a large amount of data (petabyte-scale) and compute resources (1000s of CPU cores) on an infrastructure having its own set of requirements and constraints.
+
+Our first infrastructure activity on this project, even before defining our purchase strategy, was to build a proof of concept (PoC) to validate initial designs and verify the infrastructure's capacity for scalability.
+
+As you might know, storage and compute [density grow throughout the years](https://en.wikipedia.org/wiki/Moore%27s_law), with the price per TB or per CPU core constantly decreasing. With a project planned to enter full production stage 4 years after initiation, we decided to stagger procurement throughout the project lifespan and purchase "as late as possible" with the objective of providing our users with more compute and storage resources per project-dollar spent.
+
+This approach also has an interesting side effect on long-term maintenance and warranty when comparing purchasing everything at once versus staggered purchases. Let's imagine purchasing during "Year 1", a storage server composed of 36x 4TB drives. This server will complete its 3 years warranty period during "Year 4". If we keep sufficient funds for purchases towards end of "Year 3" or beginning of "Year 4", we'll likely be in a position of purchasing storage servers composed of 36x 10TB drives. With more than double the capacity, those will be covering replacement of "Year 1" servers, while effectively providing storage components under warranty for an additional 3 years.
+
+# Available procurement methods
+
+The first step in purchasing equipment is to identify and initiate the appropriate procurement strategy. The 4 most common procurement methods available to potential buyers are [RFI, RFP, RFQ and RFSQ](https://www.negotiations.com/articles/procurement-terms/). Those methods provide a framework to purchase physical equipments (that's the most "obvious" part), but also detail all contractual elements associated with this purchase such as warranty period, support services, RMA (Return Merchandise Authorization) process, payment schedule, and so on.
+
+Although the part about equipments is relatively straight-forward, it is key to properly negotiate and identify all the contractual elements associated with hardware purchases.
+
+## RFI: Request For Information
+
+A RFI is usually used to consult vendors on the feasibility of a project and get an estimate of the associated costs. A RFI could eventually be sent to a large number of vendors and used as a mean of shortlisting vendors in preparation for a RFP or RFQ.
```

Comment:
"RFIs are usually used to consult vendors on the feasibility of a project and get an estimate of the associated costs.The RFI could eventually be sent to a large number of vendors and used as a mean of shortlisting vendors in preparation for a RFP or RFQ.